include/usgscsm/UsgsAstroLsSensorModel.h

```diff
@@ -919,22 +919,26 @@ class UsgsAstroLsSensorModel : public csm::RasterGM, virtual public csm::Settabl
    // methods pulled out of los2ecf
 
    void computeDistortedFocalPlaneCoordinates(
-       const double& lineUSGS, 
-       const double& sampleUSGS, 
-       double &distortedLine, 
+       const double& line, 
+       const double& sample, 
+       double& distortedLine, 
        double& distortedSample) const;
 
    void computeUndistortedFocalPlaneCoordinates(
-       const double& isisNatFocalPlaneX, 
-       const double& isisNatFocalPlaneY, 
-       double& isisFocalPlaneX, 
-       double& isisFocalPlaneY) const; 
+       const double& distortedFocalPlaneX, 
+       const double& distortedFocalPlaneY, 
+       double& undistortedFocalPlaneX, 
+       double& undistortedFocalPlaneY) const; 
 
    void calculateRotationMatrixFromQuaternions(
        const double& time, 
        const bool& invert, 
        double cameraToBody[9]) const;
 
+   void calculateRotationMatrixFromEuler(
+       double euler[], 
+       double rotationMatrix[]) const;

```

Comment:
This needs to be a reference to return properly?

Reply:
This is how UsgsAstroLsSensorModel::determineSensorCovarianceInImageSpace made use of it's array (sensor_cov) and successfully uses it in the same way (updating it within the method call).

Reply:
@jessemapel Doesn't this implicitly pass a pointer to the first element of the array? It's not exactly the same thing as pass by reference, but doesn't it function similarly in practice?

Reply:
@kberryUSGS That's right, I forgot about how arrays are passed around.